lib/tasks/correct_rent_type_value.rake

```diff
@@ -0,0 +1,20 @@
+desc "Alter rent_type values for bulk uploaded lettings logs for 2024 where they were not mapped correctly"
+task correct_rent_type_value: :environment do
+  affected_uploads = BulkUpload.where(log_type: "lettings", year: 2024)
```

Comment:
Is there a way to ensure that there aren't any uploads processed between the fix being deployed and running this task?

Reply:
Hmm, good point, would it make sense to add something similar to what you did with noint?

Reply:
Probably - I've been pondering if there's a better way, but can't think of anything short of turning of sidekiq for a bit which seems like a bad plan.

Reply:
Agree! 
I've updated it now to have the rent_type_fix_status. I've also removed noint_fix_status, and the job to fix noint just as a cleanup, but let me know if we still need that info and I'll revert this!

Reply:
Ooh, I just realised there's a ticket for that and it would need 2 separate releases. I'll do that instead.
I only did the first part of removing noint here, and this is removing it from the db: https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/2508

Will merge/deploy separately after this one is out